examples/run_catenary.jl

```diff
@@ -1,31 +1,43 @@
-using ControlPlots, TetherModels, StaticArrays, LinearAlgebra, KiteUtils
+using ControlPlots, TetherModels, StaticArrays, LinearAlgebra, KiteUtils, AtmosphericModels
 
-# Set initial conditions
-kite_pos = MVector{3}([100.0, 100, 800])
-tether_length = norm(kite_pos)*1.05
+# Set example kite_pos and kite_vel
+kite_pos = MVector{3}([100.0, 100.0, 300.0])
```

Comment:
Initial kite position and velocity should come from the section init of the settings.